pkg/config/config.go

```diff
@@ -865,6 +865,10 @@ func InitConfig(config Config) {
 	config.SetKnown("process_config.internal_profiling.enabled")
 	config.SetKnown("process_config.remote_tagger")
 
+	// Process Discovery Check
+	config.BindEnvAndSetDefault("process_config.process_discovery.enabled", false)
```

Comment:
on the PR description it said that the default value should be `true`. 
which value is right?

Reply:
Hi! Thanks for pointing that out. We recently decided to change the default value to `false` for this release, however it will be changed back to `true` in 7.33. I will update the description, thank you so much 😄